packages/cli/commands/sandbox/delete.js

```diff
@@ -16,8 +27,60 @@ exports.describe = i18n(`${i18nKey}.describe`);
 exports.handler = async options => {
   await loadAndValidateOptions(options);
 
-  logger.log('');
-  logger.log(i18n(`${i18nKey}.temporaryMessage`));
+  const { account } = options;
+  const config = getConfig();
+
+  let accountPrompt;
+  if (!account) {
+    accountPrompt = await deleteSandboxPrompt(config);
+  }
+  const sandboxAccountId = getAccountId({
+    account: account || accountPrompt.account,
+  });
+
+  trackCommandUsage('sandbox-delete', {}, sandboxAccountId);
+
+  let parentAccountId;
+  for (const portal of config.portals) {
+    if (portal.portalId === sandboxAccountId) {
+      if (portal.parentAccountId) {
+        parentAccountId = portal.parentAccountId;
+      } else {
+        const parentAccountPrompt = await deleteSandboxPrompt(config, true);
```

Comment:
Backwards compatibility for accounts without parentAccountId set, but will break if user selects wrong parent portal